src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/KnownOwnerViewModel.cs

```diff
@@ -10,6 +12,11 @@ public class KnownOwnerViewModel
     {
         public KnownOwnerViewModel(KnownOwner knownOwner)
         {
+            if (knownOwner is null)
+            {
+                throw new ArgumentNullException(nameof(knownOwner));
+            }
+
             Name = knownOwner.Name;
```

Comment:
Enable nullable for KnownOwner too? 
It's already pretty much done anyways.

That would justify why we don't have null checks here.

Reply:
The KnownOwner class resides in a different project with Nullable enabled in the csproj.
https://github.com/NuGet/NuGet.Client/blob/15042312732500cd8ba0369a2bd14bed5861f1bf/src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGet.VisualStudio.Internal.Contracts.csproj#L8

Reply:
oh, we have it enabled for the whole project. Nice.